pkg/api/common/helper.go

```diff
@@ -51,3 +51,8 @@ func HandleValidationErrors(e validator.ValidationErrors) error {
 	}
 	return fmt.Errorf("Namespace %s is not caught, %+v", ns, e)
 }
+
+// BoolPtr convert a bool to *bool
+func BoolPtr(b bool) *bool {
```

Comment:
there is already a function doing this at https://github.com/Azure/acs-engine/blob/7272611344e09bf7e59d565bb39457d9736511d5/pkg/helpers/helpers.go#L55 I think you could re-use it

Reply:
yep, will do